Makefile

```diff
@@ -18,13 +18,31 @@ install-deps:
 	git clone $(SNACTOR_URL) snactor
 	cd snactor && git checkout $(SNACTOR_BRANCH) && make install-deps build install
 	rm -fr snactor
+	for i in $$(find src -name 'Makefile'); do make -f $$i install-deps; done
 
 install:
-	mkdir -p $(ROOT_PATH) 
+	mkdir -p $(ROOT_PATH)
 	cp -r src/actors $(ROOT_PATH)
 	cp -r src/schemas $(ROOT_PATH)
 
 test:
-	py.test tests/*.py 
+ifeq ($(ACTOR), )
```

Comment:
add some warning if user wants to run potentially non-existent file (instead of blowing up)?

Reply:
Can you add this comment into the makefile itself? Because strictly from the code point of view, there is no reason why not to use ifneq, but if we know we wanna extend this, it will make more sense for anyone reading it, if the comment will be there